docs_src/general/PlatformRequirements.md

```diff
@@ -0,0 +1,19 @@
+# Platform Requirements
+
+EdgeX Foundry is an operating system (OS)-agnostic and hardware (HW)-agnostic IoT edge platform. At this time the following platform minimums are recommended:
+
+=== "Memory"
+    Memory: minimum of 1 GB
+=== "Storage"
+    Hard drive space: minimum of 3 GB of space to run the EdgeX Foundry containers, but you may want more depending on how long sensor and device data is to be retained.  Approximately 32GB of storage is minimally recommended to start.
```

Comment:
With Redis, this additional storage is now Memory as well, correct?

Reply:
Good point Lenny.  I took the info from the Geneva Release Notes on this an put that in the definition.